jobs/sproutsocial_platform.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Platform Engineer
+company: Sprout Social
+url: http://https://SproutSocial.applytojob.com/apply/aA9Nct/Software-Engineer-Platform.html?source=PythonJobs
```

Comment:
Hi Aaron, thanks for submitting this request.

Your URL looks a bit funny with the http://https:// bit.  I think you want this instead?

http://SproutSocial.applytojob.com/apply/aA9Nct/Software-Engineer-Platform.html?source=PythonJobs

regards